src/main/java/com/royalrangers/controller/RegistrationController.java

```diff
@@ -35,8 +35,9 @@
     private UserRepository userRepository;
 
     @PostMapping
-    public ResponseResult registration(@RequestBody String jsonUser) {
+    public ResponseResult registration(@RequestBody Map <String, Object> params) {
```

Comment:
If you have an appropriate bean for request params representation you need to use this bean. Here you'll have `@RequestBody UserBean userInfo`. From frontend side, you'll always get an object from which you'll get all fields which you need. So, there aren't any need to build such difficult structures.